node/pkg/governor/mainnet_tokens_test.go

```diff
@@ -0,0 +1,102 @@
+package governor
+
+import (
+	"github.com/certusone/wormhole/node/pkg/common"
+	"testing"
+
+	"github.com/certusone/wormhole/node/pkg/vaa"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTokenListSize(t *testing.T) {
+	tokenConfigEntries := tokenList()
+
+	/* Assuming that governed tokens will not go down over time,
+	   lets set a floor to avoid parsing or loading regressions */
+	assert.Greater(t, len(tokenConfigEntries), 122)
+}
+
+func TestTokenListFloor(t *testing.T) {
+	tokenConfigEntries := tokenList()
+
+	/* Assume that we will never have a floor price of zero,
+	   otherwise this would disable the value of the notional
+	   value limit for the token */
+	for _, tokenConfigEntry := range tokenConfigEntries {
+		testLabel := vaa.ChainID(tokenConfigEntry.chain).String() + ":" + tokenConfigEntry.symbol
+		t.Run(testLabel, func(t *testing.T) {
+			assert.Greater(t, tokenConfigEntry.price, float64(0))
+		})
+	}
+
+}
+
+func TestTokenListAddressSize(t *testing.T) {
+	tokenConfigEntries := tokenList()
+
+	/* Assume that token addresses must always be 32 bytes (64 chars) */
+	for _, tokenConfigEntry := range tokenConfigEntries {
+		testLabel := vaa.ChainID(tokenConfigEntry.chain).String() + ":" + tokenConfigEntry.symbol
+		t.Run(testLabel, func(t *testing.T) {
+			assert.Equal(t, len(tokenConfigEntry.addr), 64)
+		})
+	}
+}
+
+func TestTokenListChainTokensPresent(t *testing.T) {
+	tokenConfigEntries := tokenList()
+
+	/* Assume that all chains will have governed tokens */
```

Comment:
```suggestion
	/* Assume that all chains with a token bridge will have governed tokens */
```

Pythnet (#1459) does not have a token / nft bridge